source/plugin/text/templates.rst

```diff
@@ -0,0 +1,135 @@
+=======================
+Your first TextTemplate
+=======================
+
+TextTemplate's are an easy and convenient way to store messages with variable elements. For instance, you may find
+yourself wanting to create a configurable message for players who have joined the server for the first time. This is
+easily attainable with TextTemplates using the following strategy.
```

Comment:
Minor nitpick, `This is easily attainable with TextTemplates using the following strategy:`